tests/Moq.Analyzers.Benchmarks/Moq.Analyzers.Benchmarks.csproj

```diff
@@ -8,6 +8,7 @@
 
   <ItemGroup>
     <ProjectReference Include="$(RepoRoot)/src/Moq.Analyzers/Moq.Analyzers.csproj" />
+    <ProjectReference Include="..\..\src\Moq.CodeFixes\Moq.CodeFixes.csproj" />
```

Comment:
I don't think you should need to do this? Shouldn't it be brought in via the reference in analyzers?

Reply:
Yes, it's transitively included. I've removed it in the update for the benchmark and test projects